impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -619,6 +613,13 @@ public Map<String, Object> getConnectConfigForCluster(String bootstrapServers, S
         return kafkaConfig;
     }
 
+    /**
+     * Generates client connection config to connect to the cluster via the supplied controller bootstrap address.
+     * This method is applicable to KRaft clusters only.
+     *
+     * @param bootstrapControllers controller bootstrap
+     * @return the connect config for cluster
+     */
```

Comment:
should it throw in Zookeeper mode?

Reply:
It probably should, but KCC doesn't actually have state.  I don't really think the abstraction is quite right but the new code is in keeping with the existing code.  I'm not going to fix the deeper issue now.